### DISCUSSION TITLE
Mounting Games

### DISCUSSION THREAD

**Discussion by FrostyModder** [aldostools/webMAN-MOD#739] [Q&A]
how can i fix this the webmod makes me disable syscalls to sign in but when i do that i cant load my games says cfw syscalls already disabled

**Comment by aldostools** [comment]
You have to mount the game, then press R2+TRIANGLE to disable the syscalls.

If you enable PS3MAPI in the Pad shortcuts section of /setup.ps3, you can mount the games with the syscalls disabled. But the syscalls are not fully disabled.
